src/app/branded/branded-checkout.component.js

```diff
@@ -59,6 +64,13 @@ class BrandedCheckoutController {
   onThankYouPurchaseLoaded(purchase) {
     this.onOrderCompleted({$event: {$window: this.$window, purchase: changeCaseObject.camelCase(pick(purchase, ['donorDetails', 'lineItems']))}});
   }
+
+  fireAnalyticsEvents(...checkoutSteps){
+    checkoutSteps.forEach(checkoutStep => {
+      this.analyticsFactory.setEvent('checkout step ' + checkoutStep);
+      this.analyticsFactory.track('aa-checkout-step-' + checkoutStep);
```

Comment:
Do we need both of these still? I know Brenton was switching between them for some reason.